ipywidgets/widgets/widget_layout.py

```diff
@@ -75,6 +78,11 @@ class Layout(Widget):
     grid_column = Unicode(None, allow_none=True, help="The grid-column CSS attribute.").tag(sync=True)
     grid_area = Unicode(None, allow_none=True, help="The grid-area CSS attribute.").tag(sync=True)
 
+    def set_border(self, border):
+        """"""
+        for side in ['top', 'right', 'bottom', 'left']:
+            setattr(self, "border_" + side, border)
```

Comment:
Proposal:
 - use a property
 - make it behave like the js attribute
Setting the property would set all traits to the new value
Getting the property returns None except if all values of the traits are the same, in which case the value is returned.